src/PowerManifold.jl

```diff
@@ -1092,23 +1182,46 @@ function mid_point!(M::PowerManifoldNestedReplacing, q, p1, p2)
 end
 
 @doc raw"""
-    norm(M::AbstractPowerManifold, p, X)
+    norm(M::AbstractPowerManifold, p, X, r::REal=2.0)
 
 Compute the norm of `X` from the tangent space of `p` on an
-[`AbstractPowerManifold`](@ref) `M`, i.e. from the element wise norms the
-Frobenius norm is computed.
+[`AbstractPowerManifold`](@ref) `M`, i.e. from the element wise norms `r`-norm is computed,
+where the default `r=2.0` yields the Frobenius norm is computed.
 """
-function LinearAlgebra.norm(M::AbstractPowerManifold, p, X)
-    sum_squares = zero(number_eltype(X))
+function LinearAlgebra.norm(M::AbstractPowerManifold, p, X, r::Real = 2.0)
+    isinf(r) && return _norm_Inf(M, p, X)
```

Comment:
Technically it makes sense to distinguish positive infinity norm and negative infinity not-really-norm:
```julia
julia> norm([1, 2], Inf)
2.0

julia> norm([1, 2], -Inf)
1.0
```
So for `r` equal to `-Inf` we should either follow LinearAlgebra.jl or leave it undefined.

Reply:
Iff, this already now feels like boilerplates of boilerplates, with another case to manage – well, sure. just another 20+ lines down to hack.
And I have never seen that one in action, but sure, that can be done

Reply:
added it, though I am not sure this actually is a norm – but maybe there computer scientists (that can be done in code) and mathematicians (this is nor a norm/distance) disagree ;)

Reply:
The `-Inf` function is I think a seminorm but is definitely not a norm but in general. Well, some people seem to find it useful.

Reply:
Sure, added it already – just fighting a few remaining inverse-retractions-and-product-manifold stuff now.